tests/integration/backward_compatible/proxy/map_test.py

```diff
@@ -596,6 +617,31 @@ def test_values_with_predicate(self):
     def test_str(self):
         self.assertTrue(str(self.map).startswith("Map"))
 
+    def test_add_interceptor(self):
+        interceptor = MapGetInterceptor("hell")
+        registration_id = self.map.add_interceptor(interceptor)
+        self.assertIsNotNone(registration_id)
+
+        self.map.set(1, "o")
+        value = self.map.get(1)
+        self.assertEqual("hello", value)
+
+    def test_remove_interceptor(self):
+        skip_if_client_version_older_than(self, "5.0")
+
+        interceptor = MapGetInterceptor("hell")
```

Comment:
It would be better to use a less-provocative prefix :)

Reply:
This was the first thing that came to my mind when I saw 666 :smiling_imp: 

Anyways, I changed the texts to be nicer